pkcs11/constants.py

```diff
@@ -346,6 +346,15 @@ class CertificateType(IntEnum):
     _VENDOR_DEFINED = 0x80000000
 
 
+class Default(IntEnum):
+    """
+    Wraps DEFAULT so we don't accidentally coincide
+    with a desired value of 0xFFFFFFFF
+    """
+    DEFAULT = 0xFFFFFFFF
```

Comment:
Use the [sentinel object pattern](https://python-patterns.guide/python/sentinel-object/#sentinel-objects). `IntEnum` is especially dangerous because 0xffffffff might be a valid value for something (indeed I think it is for mantissa).

```python
DEFAULT = object()

if value is DEFAULT:
    ....
```